### PR TITLE
[Issue 514] Remove NOFO content from static site milestone

### LINKED ISSUE #514: [Task]: Remove NOFO references from Static Site milestone doc
### Summary

Update [Static Site milestone doc](https://github.com/HHS/grants-equity/blob/main/documentation/milestones/individual_milestones/static_site.md) to remove references to NOFO Transformation workstream content

### Acceptance criteria

- [ ] References to the NOFO transformation workstream have been removed from the milestone doc
- [ ] The issue in GitHub has also been updated to reflect this change

### PR DESCRIPTION
**Pull request by widal001**
## Summary

Updates milestone docs to remove NOFO content and target release date attribute

Fixes #514 

### Time to review: __3 mins__

- Removes references to NOFO transformation workstream from milestone doc
- Removes "target release date" row from milestone doc metadata
- Updates template to include metadata that is consistent across milestone docs

It may be easier to view the rendered versions of the docs to review

### CODE REVIEW

File: documentation/milestones/individual_milestones/static_site.md [hunk 1]
```diff
@@ -105,12 +103,11 @@ Process for drafting and updating the content of the site should balance:
...
 6. Deployment build time
 7. Deployment/hosting costs
-8. Number of PDF views per NOFO prototype
-9. Number of visits to outbound links to the following external resources (once added to the site)
+8. Number of visits to outbound links to the following external resources (once added to the site)
    - Open source repository
    - Open source wiki
    - Open source chat
-10. Stretch: Number of form responses
+9.  Stretch: Number of form responses
```
Comment (andycochran) [inline documentation/milestones/individual_milestones/static_site.md:110]: Not asking for a change here but wanted to note a PROTIP: 

> To create an ordered list, add line items with numbers followed by periods. The numbers don’t have to be in numerical order, but the list should start with the number one.

So you can do this: 
```
1. First item
1. Second item
1. Third item
1. Fourth item
```
…and it'll render like this: 
1. First item
1. Second item
1. Third item
1. Fourth item

…then you don't have to worry about setting the right number. 

Aside: This reminds me of [an article](https://siderea.dreamwidth.org/1819759.html) I just read about "a single HTML tag (ordered lists) and end with the downfall of civilization."

Comment (widal001) [reply documentation/milestones/individual_milestones/static_site.md:110]: @andycochran This is a good tip to limit unnecessary diffs!

Currently I use [Markdown All in One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one) as my VSCode extension for formatting Markdown, which automates the process of updating numbers in ordered lists, so (fortunately) I'm not manually reordering these myself. Still it is a pain to see the diffs on lines that didn't actually result in content changes.

I'll have to check if that VSCode extension supports configuring ordered lists to only use `1.` like you mentioned. Do you use a different markdown extension for VSCode?

Comment (widal001) [reply documentation/milestones/individual_milestones/static_site.md:110]: Also that article is a _wild_ ride, but I love it!! I could feel [the computational law](https://law.stanford.edu/2021/03/10/what-is-computational-law/) recesses of my brain itching as I was reading it 😅

Comment (andycochran) [reply documentation/milestones/individual_milestones/static_site.md:110]: Ooh! I'll check out the Markdown All in One extension.

Comment (widal001) [reply documentation/milestones/individual_milestones/static_site.md:110]: _Same_!! Atom was so great, I'm still salty they sunsetted it. And Atom did have a great Markdown plugin that followed the `1.` only style for ordered lists

**Pull request merged**